src/cache/CacheResultProcessor.ts

```diff
@@ -1,6 +1,6 @@
 import _ from "lodash";
 import { FieldNode, SelectionNode } from "graphql";
-import { Cache, makeReference } from "@apollo/client";
+import { Cache, makeReference } from "@apollo/client/core";
```

Comment:
So was the issue that by importing from '@apollo/client` we were requiring some react dependencies which now that we only import from cache, we can remove the react dev dependency?

Reply:
Exactly! the `@apollo/client` is heavily biased towards react, by using the core we only use it if it's necessary